dashboard/app/controllers/projects_controller.rb

```diff
@@ -15,6 +15,9 @@ class ProjectsController < ApplicationController
     playlab: {
       name: 'New Play Lab Project'
     },
+    starwars: {
+      name: 'New Star Wars Project'
```

Comment:
Are we okay with using this name?  (The tutorial's full name was `Star Wars: Building a Galaxy With Code` and we had to use that pretty consistently.)